---
layout: page
title: About
permalink: /about/
---

![_config.yml]({{ site.baseurl }}/images/lab_website.png)

I am currently a postdoc in [Dr. Rob Knight's lab](https://knightlab.ucsd.edu) (see also [here](https://www.nae.edu/312487/Dr-Rob-Knight)) at University of California, San Diego. My research focus is computational Microbiology, including, but not limited to, computational biology, multi-omics, data structures, high performance computing, environmental microbiology and microbial ecology. I obtained my Ph.D in bioinformatics (minor in Computational Science and Engineering) (2024) at [Center for Bioinformatics and Computational Genomics](https://bioinfo.gatech.edu), Georgia Institute of Technology, supervised by [Dr. Konstantinos T Konstantinidis](https://scholar.google.com/citations?user=cT7dxlYAAAAJ&hl=en&oi=sra). I obtained my Master degree in Environemntal Science and Engineering (2019) from Tsinghua University, Beijng, China, under the superversion of [Dr. Jizhong Zhou](https://www.ou.edu/cas/mpbio/people/faculty/zhou) (see also [here](https://www.nasonline.org/directory-entry/jizhong-zhou-nwkro9/)). I am very insterested in designing and implementing data structures and algorithms to solve computational problems with respect to microbial genomes. Being the most diverse group on earth, tons of microbes (e.g. bacteria, phages) have been sequenced and more than 80% of the genomic sequences in NCBI are microbial genome sequences. A total of more than 150T microbial genomic sequences has been deposited and it is growing at a speed much faster than the development of transistors. Handling those huge amount of genomic sequences from microbes requires smart data structures, efficient algorithms and high performance computing (e.g., massively parallel computing). My research aim is to combine algorithm & data structure development, parallel and concurrent computing/programming with genome informatics to solve problems in large-scale genome information mining.

I am a Rustacean (锈真菌生物,my own translation, a user of the Rust programming language)! I am a fan of parallel and concurrent programming. Go Ferris!

![_config.yml]({{ site.baseurl }}/images/Ferris_panicked.PNG)

### Key Publication

#### Under Review & Preprint

•	Zhao J, Both JP, Knight R. Ultra-fast and efficient network embedding for gigascale biological datasets. bioRxiv. 2025:2025.06.18.660497 [Link](https://doi.org/10.1101/2025.06.18.660497)

•	Zhao J, Zhao X, Pierre-Both J et al. Bindash 2.0: New minhash scheme allows ultra-fast and accurate genome search and comparisons. bioRxiv. 2024:2024.03.13.584875 [Link](https://www.biorxiv.org/content/10.1101/2024.03.13.584875v1)

•   Oles R. et.al. 2025. Comparative genomic analysis of gut-resident and extra-intestinal Bacteroides fragilis strains. 

#### Published or accepted 
##### 2025

•   Aton et.al.. Scikit-bio: a fundamental Python library for biological omic data analysis, Nature Methods, 2025. Accecpted, link to come

•	Jianshu Zhao, Genevieve Brandt, Jessica L Gronniger, Zhao Wang, Jiaqian Li, Dana E Hunt, Luis M Rodriguez-R, Janet K Hatt, Konstantinos T Konstantinidis, Quantifying the contribution of the rare biosphere to natural disturbances, The ISME Journal, 2025;, wraf129, https://doi.org/10.1093/ismejo/wraf129

***This paper introduces a new idea to microbial ecology: using population genomes reconstructed from metagenomes to test ecological theories, such as the insurance hypothesis and the specialization–disturbance hypothesis***. 


•	Zhao, J., Pachiadaki, M., Conrad, R.E., Hatt, J.K., Bristow, L.A., Rodriguez-R, L.M., Rossello-Mora, R., Stewart, F.J. and Konstantinidis, K.T., 2025. Promiscuous and genome-wide recombination underlies the sequence-discrete species of the SAR11 lineage in the deep ocean. The ISME Journal, p.wraf072. https://academic.oup.com/ismej/advance-article/doi/10.1093/ismejo/wraf072/8115880 

***This paper studied the abundant marine population SAR11 and the underlying cohesive forces of species formation--homogonous recombination. SAR11 seems to adopt a very different recombination mechanism, approximating the lowest possible bound to be efficient***. 

•	Zhu, C., Wu, L., Ning, D., Tian, R., Gao S., Zhang B., Zhao J. et al. Global diversity and distribution of antibiotic resistance genes in human wastewater treatment systems. Nat Commun 16, 4006 (2025). https://doi.org/10.1038/s41467-025-59019-3 

##### 2024
•	Zhao, Jianshu., Both, J.P., Rodriguez-R, L.M. and Konstantinidis, K.T. (2024) GSearch: ultra-fast and scalable genome search by combining K-mer hashing with hierarchical navigable small world graphs. Nucleic Acids Research, 52, e74-e74.  https://academic.oup.com/nar/article/52/16/e74/7714450 

***This paper once and for all solved the large-scale genome search and classification problem.***

•	Jianshu Zhao, Jean Pierre Both, Konstantinos T Konstantinidis, Approximate nearest neighbor graph provides fast and efficient embedding with applications for large-scale biological data, NAR Genomics and Bioinformatics, Volume 6, Issue 4, 2024, lqae172, https://doi.org/10.1093/nargab/lqae172

***This paper greatly speedup non-linear dimension reduction and clustering for large-scale (e.g., billions) biological data including genomic data, single-cell RNA sequencing data.***

•	Qi, Q., Ning, S., Guo, X., Zhao, J., Tian, R., Gui, H., He, J.-S., Wang, H., Zhang, Z., Konstantinidis, K. T., Gao, Q., Wang, Y., Li, S., Zhao, W., Yang, Y., & Zhou, J. (2024). More sensitive microbial responses to the interactive effects of warming and altered precipitation in subsoil than topsoil of an alpine grassland ecosystem. Global Change Biology, 30, e17487. https://doi.org/10.1111/gcb.17487

##### 2022
•	Menezes, O., Kocaman, K., Wong, S., Rios-Valenciana, E.E., Baker, E.J., Hatt, J.K., Zhao, Jianshu., Madeira, C.L., Krzmarzick, M.J., Spain, J.C. and Sierra-Alvarez, R., 2022. Quinone Moieties Link the Microbial Respiration of Natural Organic Matter to the Chemical Reduction of Diverse Nitroaromatic Compounds. Environmental Science & Technology, 56(13), pp.9387-9397.

•	Qi, Qi, Jianshu Zhao, et al. Microbially enhanced methane uptake under warming enlarges ecosystem carbon sink in a Tibetan alpine grassland. Global Change Biology 28, 6906-6920 (2022)

##### 2021
•	Qi, Qi, Yue Haowei, Zhenhua Zhang, Joy D. Van Nostrand, Linwei Wu, Xue Guo, Jiajie Feng, Mengmeng Wang, Sihang Yang, Jianshu Zhao. et al. Microbial Functional Responses Explain Alpine Soil Carbon Fluxes under Future Climate Scenarios. mBio 12, e00761-00720 (2021)

##### 2020
•	Wan, X., Gao, Q., Zhao, J., Feng, J., van Nostrand, J.D., Yang, Y. and Zhou, J., 2020. Biogeographic patterns of microbial association networks in paddy soil within Eastern China. Soil Biology and Biochemistry, 142, p.107696.

##### 2019
•	Zhao J., Gao Q., Zhou J. et.al., 2019. The scale dependence of fungal community distribution in paddy soil driven by stochastic and deterministic processes. Fungal Ecology 42 (2019) 100856


•	Gao J., Zhao J., et.al., 2019. The spatial scale-dependency of diazotrophic and bacterial community assembly in paddy soil. Global Ecology and Biogeography. 28:1093-1105

##### 2015
•	Yan Huang, Li Sun, Jianshu Zhao, et.al., 2015. Utilization of different waste proteins to create a novel PGPR-containing bio-organic fertilizer. Scientific Report. 5(1), pp.1-11.


•	Zongzhuan Shen, Shutang Zhong, Jianshu Zhao et.al., 2015. Effects of ammonia fumigation on soil microflora and banana production in an orchard with serious Fusarium wilt disease. Acta Ecologica Sinica, 35(9), pp.2946-2953




### More Information

During my Master, I also visited Norwegian University of Life Science in Norway (2018), studying biogeochemistry, under the supervison of [Dr. Jan Mulder](https://www.nmbu.no/en/about/employees/jan-mulder) (Norwegian Academy of Sciences). I did my undergraduate theis with [Dr. Qirong Shen](http://faculty.njau.edu.cn/shenqirong/en/index.htm) (National Academy of Engineering, China).


### Contact me

[jianshuzhao@yahoo.com](mailto:jianshuzhao@yahoo.com) or [jiz322@ucsd.edu](mailto:jiz322@ucsd.edu)
